satellite/audit/reservoir_test.go

```diff
@@ -0,0 +1,29 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package audit
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"storj.io/storj/pkg/storj"
+)
+
+func TestReservoirSampling(t *testing.T) {
+	reservoir := NewReservoir(1)
+
+	path1 := storj.Path("test/path1")
+	path2 := storj.Path("test/path2")
+	path3 := storj.Path("test/path3")
+
+	reservoir.sample(path1)
+	require.Equal(t, 1, len(reservoir.Paths))
+
+	reservoir.sample(path2)
+	reservoir.sample(path3)
+	require.Equal(t, 1, len(reservoir.Paths))
+
+	// todo: ... not really sure how/what to test at this level
```

Comment:
We should also have a test for a reservoir with size > 1. At the end, we can check that there are no duplicated items in the reservoir.